Backpacks.cs

```diff
@@ -581,6 +591,63 @@ private bool VerifyCanOpenBackpack(BasePlayer looter, ulong ownerId)
             return true;
         }
 
+        void OnPlayerConnected(BasePlayer player)
+        {
+            CreateGUI(player);
+        }
+
+        void OnPlayerSleepEnded(BasePlayer player)
+        {
+            CreateGUI(player);
+        }
+
+        void CreateGUI(BasePlayer player)
+        {
+            if (player == null || (player is NPCPlayer) || (player is HTNPlayer))
```

Comment:
I think the issue you had was it should be `player.IsNpc` (capital i) not `player.isNpc`. It's part of `BaseEntity` so all entities will have it.

Reply:
Well, copypasted just like was in your post :)